Glossario.json

```diff
@@ -294,6 +294,11 @@
     "acronyms": ["PEV"],
     "synonyms": []
   },
+  "Project Planned Value": {
+	"description": "Valore pianificato per l'intero progetto in un preciso istante di tempo.",
```

Comment:
Aggiungere dopo il punto " Il suo valore è determinato dalla somma del valore pianificato per ogni periodo di sviluppo fino all'istante di tempo preso in considerazione.""